### ISSUE TITLE
freenect 1:0.1.2 breaks ntk/camera/grabber interface (doesn't build)

### ISSUE THREAD

**Issue by ghost** [rgbdemo/nestk#17]
I've recently rolled back from freenect 1:0.1.2 (neurodebian repo) to the less current freenect 1:0.0.1 (ubuntu precise). Nestk wouldn't build anymore halting on the 'ntk/camera/grabber'.

Skimming through the code my preliminary conclusion was that the grabber interface in the freenect back-end has changed and some methods aren't present anymore.

thoughts on this?

greets, thanks, Luuk

**Comment by nburrus** [comment]
Yes, supporting recent freenect would require some changes, since I think
they broke the API. It should not be so painful though.
